cmd/skywire-visor/commands/root.go

```diff
@@ -129,6 +156,7 @@ var rootCmd = &cobra.Command{
 		// --all unhide flags and print help menu
 		if all {
 			for _, j := range hiddenflags {
+				fmt.Println(j)
```

Comment:
Line left over from testing?

Reply:
yes, I was troubleshooting the hidden flags, thanks for pointing that out

Reply:
removed here

[692104f1133644bbe696e037f8fb7bb799b77a13](https://github.com/skycoin/skywire/commit/692104f1133644bbe696e037f8fb7bb799b77a13)